settings.gradle

```diff
@@ -1,2 +1,3 @@
-include 'demo-assethelper', 'demo-sqliteprovider'
-
+include 'demo-assethelper'
+include 'demo-sqliteprovider'
+include 'analyzer'
```

Comment:
@frapontillo If we add it as a regular module, that breaks the build. 

The problem is that a Gradle plugin is a buildscript dependency. As such it is resolved before any regular modules. Usually you'd need to deploy a new version of the plugin to a local repository and then let your samples depend on that. We managed to directly link the sources via some wiring in the `buildSrc` folder. Not pretty, but much faster than having to deploy each time.